### ISSUE TITLE
Is there a pytorch version of git loss?

### ISSUE THREAD

**Issue by wangjue-wzq** [kjanjua26/Git-Loss-For-Deep-Face-Recognition#3]
Is there a pytorch version of git loss?When I use gitloss for scene classification, the gitloss value does not decline, I don't know if it is a problem I implemented or gitloss itself is not suitable for scene classification.

**Comment by kjanjua26** [comment]
Are you using gitloss with softmax or are you solely training it?

**Comment by kjanjua26** [comment]
Hey,
Are you still facing the same issue? Did you try coupling softmax with Gitloss? If the issue is solved, please close the issue.
